src/Modules/UngradedModules/FreeModElem.jl

```diff
@@ -147,10 +154,11 @@ end
 Return the standard basis of `F`.
 """
 function basis(F::AbstractFreeMod)
-  bas = elem_type(F)[]
+  bas = Vector{elem_type(F)}(undef, dim(F))
+  e = one(base_ring(F))
   for i=1:dim(F)
-    s = Hecke.sparse_row(base_ring(F), [(i, base_ring(F)(1))])
-    push!(bas, F(s))
+    s = sparse_row(base_ring(F), [i], [e]; sort=false)
```

Comment:
```suggestion
    s = sparse_row(base_ring(F), [i], [e])
```
no longer needed due to https://github.com/thofma/Hecke.jl/pull/1502